Musico/app/models/group_session.rb

```diff
@@ -0,0 +1,5 @@
+class GroupSession < ApplicationRecord
+  has_and_belongs_to_many :users , dependent: :destroy
+  #admin who created session
+  belongs_to :user 
```

Comment:
May be call it group_admin, it will help in understanding
```suggestion
  belongs_to :group_admin, class_name: 'User', foreign_key: :user_id
```